syftlib/src/main/java/org/openmined/syft/networking/clients/SyftWebSocket.kt

```diff
@@ -60,6 +66,11 @@ internal class SyftWebSocket(
      */
     private var webSocket: WebSocket? = null
 
+    /**
+     * Amount of delay before attempt to reconnect
+     */
+    private var retryDelay = 1000L
```

Comment:
This should be local variable in onFailure as whenever error occurs, it should start the exponential backoff from starting i.e. 1000L